agent/bench-scripts/pbench-user-benchmark

```diff
@@ -62,6 +67,20 @@ while true; do
 			shift;
 		fi
 		;;
+                --sysinfo)
+                shift;
+                if [ -n "$1" ]; then
+                       sysinfo="$1"
+                       shift;
+                fi
+                ;;
+		--check)
```

Comment:
I am a little confused as to what the check is for.  If it's simply to get a list back on what options are available, then I'd rather have a --help option for pbench-collect-systinfo provide those in the output.